lib/cli/run.js

```diff
@@ -258,15 +259,19 @@ exports.builder = yargs =>
 
       // yargs.implies() isn't flexible enough to handle this
       if (argv.invert && !('fgrep' in argv || 'grep' in argv)) {
-        throw new Error(
-          '"--invert" requires one of "--fgrep <str>" or "--grep <regexp>"'
+        throw createMissingArgumentError(
+          '"--invert" requires one of "--fgrep <str>" or "--grep <regexp>"',
+          '--invert',
+          argv.invert
```

Comment:
Think this should be the "expected argument datatype" (see https://github.com/mochajs/mocha/blob/master/lib/errors.js#L73) ..so I think "string" would be fine

Reply:
yep. `string|regexp`